addons/project/static/src/js/tours/project.js

```diff
@@ -162,25 +162,34 @@ registry.category("web_tour.tours").add('project_tour', {
     position: "bottom",
     run: "click",
 },
-
 {
     trigger: ".o_form_project_tasks",
     isActive: ["auto"],
 },
 {
+    isActive: ["auto"],
     trigger: ".o_field_widget[name='user_ids'] input",
     content: _t("Assign a responsible to your task"),
     position: "right",
-    run: "edit Mitch",
+    run: "edit Dummy && clear",
```

Comment:
Why not simply creating a user in `test_01_project_tour` before starting the tour? Instead of doing that magic thing. 🤔

Reply:
Hey @xavierbol , I agree about creating a user in ```test_01_project_tour``` what about running automatic tour from debug menu without demo data?

What do you think?

Reply:
Hello, when we automatically run the tour, it means we have to debug it because there is an issue and so before starting it, we will have to create the user.

Reply:
Hey @xavierbol,

I was taking about this [case](https://drive.google.com/file/d/1FDvZ0t738sZtPHq1ACErsO-veRil6vui/view?usp=sharing).Have a look.Also to startup this we need to delete some steps from the tour because they were conflicting and failing the tour prematurely(only in this mode presumably or is headless testing(from start_tour) too fast(runs at 0ms delay) that it skips some steps.). All the tours would need to be refactored.

Or is this a case that we can neglect. I think we are calling different modes of tours running by different names and this is why I made the video.

What do you think?

Reply:
Why the tour has to be refactored? 

In 17, it works, right? If yes, then the recent changes introduced a limitation, IMO.

Reply:
Everything works, but we can refactor in saas-17.4 and master because many new steps like to check extra triggers(empty tour step with just a trigger IMO existing triggers can be fortified so that we don't need extra triggers in places where it is possible) were added and some steps have been duplicated due to recent changes in tour engine. I mean like not the logic or flow of steps but some dummy steps that can be removed and better logic in those steps can be used.

Reply:
May be now is not time or we could wait till whole tour engine task to be finished.

Reply:
> May be now is not time or we could wait till whole tour engine task to be finished.

yep indeed, I agree.

If now with your changes, the tour works in python test and also when we manually start it (onboarding tour) then it is good enough for me.

Reply:
It works in both methods you mentioned above with the current solution(The case I mentioned will run too if the tour engine is completed). But with ```run: "edit Mitch" ``` we would need to create data in the python test case(to make work it out without demo data) and case I mentioned above fails. Which path would you want me to proceed in ?

![guide-me-to-the-light-saint-bodhi](https://github.com/user-attachments/assets/f31aa568-fa27-41a7-8721-bde67442c590)

Reply:
The python test should not fail without demo data, so we have to change that to be sure the test is not dependant to demo data.

Reply:
Hey @xavierbol, I had a big brain moment right now if we put the trigger as 
```run: "edit Admin"``` Then it would be able to target both Mitchell Admin(With demo data) and Administrator(without demo data). Considering the other nuances such as any changes in demo data or new version of demo data would completely destroy this. Just give me an thumbsup if you are okay or else I have code ready that creates user in test case. I will push that and you can have a look. Thanks for the patience.

Reply:
Also the only place that tour dependent on demo data is while selecting the user(Will double check :zap:).

Reply:
> Hey @xavierbol, I had a big brain moment 🧠 right now if we put the trigger as `run: "edit Admin"` Then it would be able to target both Mitchell Admin(With demo data) and Administrator(without demo data). Considering the other nuances such as any changes in demo data or new version of demo data would completely destroy this. Just give me an thumbsup if you are okay or else I have code ready that creates user in test case. I will push that and you can have a look. Thanks for the patience.

indeed we can do that, good idea

Reply:
Hello @xavierbol I have pushed the code and branch is green. There seemed to be a runbot error that would be solved in this I have put it in unverified section in the task. Please have a look when you are free. Have a nice day